src/main/kotlin/racingcar/Application.kt

```diff
@@ -1,5 +1,107 @@
 package racingcar
 
+import camp.nextstep.edu.missionutils.Randoms
+import racingcar.view.InputView
+
 fun main() {
     // TODO: 프로그램 구현
+    val inputView = InputView()
+    val carNamesInput = inputView.getInputCarNames()
+    validateCarNamesInput(carNamesInput)
+
+    val splitCarNames = splitCarName(carNamesInput)
+    val roundInput = inputView.getInputRound()
+    val round = stringToInt(roundInput)
+
+    val carInstances = createCars(splitCarNames)
+    val racingGame = RacingGame(round, carInstances)
+    racingGame.play()
+    val winners = racingGame.getWinners()
+
+    printOutput(winners)
+}
+
+fun createCars(splitCarNames: List<String>): List<Car> = splitCarNames.map { Car(it) }
+
+fun validateCarNamesInput(carNamesInput: String) {
+    require(carNamesInput.isNotBlank()) { "이름이 입력되지 않았습니다." }
+    val hasInvalidChar = carNamesInput.any { it != ',' && !it.isLetter() }
+    require(!hasInvalidChar) { "이름을 올바르게 입력해주세요." }
+}
+
+fun splitCarName(carNamesInput: String): List<String> {
+    val splitCarNames = carNamesInput.split(",")
+    val hasLengthFiveOrLess = splitCarNames.all { it.length <= 5 && it.isNotBlank() }
+    require(hasLengthFiveOrLess) { "이름은 5글자를 초과하거나 비어있을 수 없습니다." }
+
+    return splitCarNames
+}
+
+fun stringToInt(roundInput: String): Int = if (roundInput.isNotBlank()) {
+    roundInput.toIntOrNull() ?: throw IllegalArgumentException("숫자를 입력해주세요.")
+} else {
+    0
+}
```

Comment:
빈 값을 입력하면 0으로 반환하도록 처리하셨는데,
이렇게 되면 게임 결과 계산 시 모든 유저가 우승으로 처리될 것 같습니다.
해당 케이스를 에러로 처리하지 않고 0을 반환하도록 하신 이유가 궁금합니다!

또한 가독성을 위해서 명시적으로 return을 사용하는 형태로 바꾸면 더 읽기 쉬울 것 같습니다!
```suggestion
fun stringToInt(roundInput: String): Int {
    if (roundInput.isNotBlank()) {
        return roundInput.toIntOrNull() ?: throw IllegalArgumentException("숫자를 입력해주세요.")
    }
    return 0
}
```

Reply:
> 빈 값을 입력하면 0으로 반환하도록 처리하셨는데, 이렇게 되면 게임 결과 계산 시 모든 유저가 우승으로 처리될 것 같습니다. 해당 케이스를 에러로 처리하지 않고 0을 반환하도록 하신 이유가 궁금합니다!

이건 제 요구사항 분석 과정에서 잘못된 판단이 들어간 것 같습니다. 말씀하신 대로 round 입력이 0이거나 들어오지 않을 경우에는 모종의 이유로 게임 개최는 했지만 진행할 수 없는 상황이라고 판단하고 모든 유저가 우승자가 되도록 구현했습니다.

지금 생각해보니 IllegalArgumentException을 발생시키거나 우승자가 없도록 하는 것이 더 올바른 구현일 것 같네요.